res/res_respoke_session.c

```diff
@@ -819,6 +820,28 @@ int respoke_session_offer(struct respoke_session *session)
 
 int respoke_session_answer(struct respoke_session *session)
 {
+	if (session->channel)
```

Comment:
Coding guidelines for Asterisk modules typically will put the { on the same line as the if statement:

```
if (session->channel) {
```